lime-cli/src/main/scala/org/bdgenomics/lime/cli/Jaccard.scala

```diff
@@ -0,0 +1,83 @@
+package org.bdgenomics.lime.cli
+
+import org.apache.spark.SparkContext
+import org.bdgenomics.adam.rdd.ADAMContext._
+import org.bdgenomics.adam.rdd.{ ADAMSaveAnyArgs, GenomicRDD, feature }
+import org.bdgenomics.lime.set_theory.DistributedIntersection
+import org.bdgenomics.lime.set_theory.DistributedMerge
+import org.bdgenomics.utils.cli._
+import org.kohsuke.args4j.Argument
+import org.apache.spark.storage.StorageLevel
+import org.bdgenomics.formats.avro.Feature
+import org.bdgenomics.adam.models.ReferenceRegion
+import org.bdgenomics.adam.rdd.feature.FeatureRDD
+object Jaccard extends BDGCommandCompanion {
+  val commandName = "jaccard"
+  val commandDescription = "Compute jaccard distance between two inputs"
+
+  def apply(cmdLine: Array[String]) = {
+    new Jaccard(Args4j[JaccardArgs](cmdLine))
+  }
+
+  class JaccardArgs extends Args4jBase with ADAMSaveAnyArgs with ParquetArgs {
+    @Argument(required = true,
+      metaVar = "INPUT1",
+      usage = "The first file for Jaccard",
+      index = 0)
+    var leftInput: String = null
+    @Argument(required = true,
+      metaVar = "INPUT2",
+      usage = "The second file for Jaccard",
+      index = 1)
+    var rightInput: String = null
+
+    override var sortFastqOutput: Boolean = false
+    override var asSingleFile: Boolean = false
+    override var deferMerging: Boolean = false
+    override var outputPath: String = ""
+  }
+
+  class Jaccard(protected val args: JaccardArgs) extends BDGSparkCommand[JaccardArgs] {
+    val companion = Jaccard
+
+    def run(sc: SparkContext) {
+      val leftGenomicRDD = sc.loadBed(args.leftInput).repartitionAndSort()
+      val leftGenomicRDDKeyed = leftGenomicRDD.rdd.map(f => (ReferenceRegion.unstranded(f), f))
+      val rightGenomicRdd = sc.loadBed(args.rightInput).repartitionAndSort()
+      val rightGenomicRDDKeyed = rightGenomicRdd.rdd
+        .map(f => (ReferenceRegion.unstranded(f), f))
+
+      //Intersection of the two files and merging the result to calculate intersect_lengthe
+      val intersectedRdd = DistributedIntersection(leftGenomicRDDKeyed, rightGenomicRDDKeyed, leftGenomicRDD.partitionMap.get)
+        .compute().map(f => {
```

Comment:
Why are you mapping over the results here and creating the new feature? The jaccard statistic doesn't require a feature, right?